gen3.datacommons.io/manifest.json

```diff
@@ -222,7 +222,7 @@
       "serviceAccountName": "jobs-gen3-datacommons-io",
       "container": {
         "name": "job-task",
-        "image": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifest-merging:2023.10",
+        "image": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/cdis/manifest-merging:4.0.1",
```

Comment:
i don't think this is right?

Reply:
I was just copying what @k-burt-uch did [here](https://github.com/uc-cdis/cdis-manifest/blob/871646de870cb0368b05505f42727c36264f1cf8/preprod.gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json#L301)  for BDC

Reply:
looks like there was a preexisting typo: https://github.com/uc-cdis/cdis-manifest/commit/871646de870cb0368b05505f42727c36264f1cf8#diff-f74e04563308c2dffcb086dd48b8f41748d50088364cc7dbf717e32657ae4381L301

Reply:
whoops - sorry about that!